Update joining-the-testnet for The Quest (#171)
oasislabs/docs: src/operators/joining-the-testnet.md

# Joining the Testnet

This guide will cover setting up your nodes for The Quest, a multi-round staking
competition (aka incentivized testnet) on the Oasis Network. It assumes some basic
knowledge on the use of command line tools.

::: tip NOTE

> If you joined the Testnet prior to 1/15 and you plan to use the same entity for

Author: ```suggestion
If you joined the Testnet prior to 2019/01/15 and you plan to use the same entity for
```

> If you joined the Testnet prior to 2019/01/15 and you plan to use the same entity for

Author: ```suggestion
If you joined the Testnet prior to 2019/01/15 and you plan to use the same entity
```

> The Quest, use the following steps to upgrade:

Author: ```suggestion
for The Quest, use the following steps to upgrade:
```

## Signing up for Testnet Tokens

::: tip NOTE
This step is not necessary if you submitted a valid entity package for The Quest

> prior to the start on 1/15--in this case, your entity will be included in the

Reviewer: ```suggestion
prior to 2019/01/13 @ 00:00 UTC--in this case, your entity will be included in the
```

> prior to 2019/01/13 @ 00:00 UTC--in this case, your entity will be included in the

Author: ```suggestion
prior to 2019/01/13 @ 00:00 UTC. In this case, your entity will be included in the
```
